Derivative.py

```diff
@@ -39,10 +39,10 @@ def _md4_impl_(dx, numerical_flux, center_flux, order_used):
     for j in range(len(numerical_flux)):
         temp = np.zeros(len(numerical_flux[j]) - 1)
         for i in range(1, len(numerical_flux[j]) - 2):
-            temp[i] = 9 / 8 * (numerical_flux[j][i + 1] -
-                               numerical_flux[j][i]) / dx - 1 / 24 * (
-                                   numerical_flux[j][i + 2] -
-                                   numerical_flux[j][i - 1]) / dx
+            temp[i] = 9.0 / 8.0 * (numerical_flux[j][i + 1] -
```

Comment:
Just an FYI I learned about Python... Precomputing these ratios makes a surprisingly large speed difference. So when you see long numbers throughout the code it's because that was much faster than having the fraction....